tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -941,9 +941,30 @@ const reducer = createReducer(
     };
   }),
   on(actions.selectTimeEnableToggled, (state) => {
+    let nextCardStepIndexMap = {...state.cardStepIndex};
+    const nextSelectTimeEnabled = !state.selectTimeEnabled;
+    const {min} = state.stepMinMax;
+    const startStep = min === Infinity ? 0 : min;
+    const nextSelectedTime = state.selectedTime ?? {
```

Comment:
Would it be wise to only initialize nextSelectedTime if nextSelectTimeEnabled? I realize that, right now, in practice, the first time selectTimeEnableToggled is fired it is because the user is turning on linked time BUT it would be good to be certain.